src/components/Header/index.scss

```diff
@@ -1,16 +1,35 @@
-// Header top background-color #340EC1
-// Header search background-color #4D25E0
+@mixin mQ($arg) {
```

Comment:
![Screen Shot 2021-06-20 at 9 28 07 AM](https://user-images.githubusercontent.com/11296684/122665841-0268cc00-d1aa-11eb-9330-c6ca837d4ac2.png)
- general UI issues:
1. Order of elements is not correct in screen sizes larger than 320px width it should still having the same order in all mobile sizes like iPhone 6,7,8 iPhone 6,7,8 plus, iPhone X , Galaxy s5,6 and pixel 2,2XL and Moto4G.

- Devices <= 320px issues
1. All products menu button should be full width and also search input and search input should be above all products button also menu items should be full with with expand and collapse effect.
2. Other drop downs (account and language) should have the same effect of collapse and expand and they should be full width.

- iPad Pro issues (1024*1366)
1. `black btn` text is on 2 lines.
2. logo and all products menu btn has no margin between them.
3. `all products` text is on 2 lines .
4. there is a big blue empty space at the bottom of `hero view secion`.
5. there is no vertical spacing between hero view elements.
6. `Shop now button` height is bigger than normal in categories section and also the categories card has lots of empty vertical space.
7. For discount card elements should be `one element on each line` even the `rate count` with good `spacing between elements` to fix `card height`.
8. Fix spacing between discount cards.
9. there is a big white empty space between promotions and discounts.
10. `cart count` has no margin between it and cart word.

- Issues in screens > 320px width and < iPad (768px)
1. All products menu button should be full width and also search input and search input should be above all products button also menu items should be full with with expand and collapse effect.
2. top header laber and btn should be on 2 lines 
3.subscribe button text is larger than it should be making it goes on 2 lines.
4. Hero view elements is not laid out properly.
5. categories card text size is larger than it should be and image is not showing properly.
6. discount card elements are not laid out properly.
7. promotion card in the middle has issue in `browse products button` alignment.
8. Other drop downs (account and language) should have the same effect of collapse and expand and they should be full width.